public/editor/scripts/project/project.js

```diff
@@ -1,4 +1,5 @@
 define(function(require) {
+  var $ = require("jquery");
```

Comment:
Hmm... I would recommend we stop doing that, but if we want to keep this PR in line with the rest of the code that's fine with me.